java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -282,4 +282,41 @@ public interface ServerManagementClusterCommands {
      * }</pre>
      */
     CompletableFuture<ClusterValue<String[]>> time(Route route);
+
+    /**
+     * Saves the DataBase in background.<br>
+     * The command will be routed to a random node.
+     *
+     * @see <a href="https://redis.io/commands/bgsave/">redis.io</a> for details.
+     * @param schedule Flag to schedule save on the next opportunity.
+     * @return A server confirmation whether background save started or scheduled.<br>
+     *     An error is returned if there is already a background save running
+     * @example
+     *     <pre>{@code
+     * String response = client.bgsave().get();
+     * assert response.equals("Background saving started")
+     *     || response.equals("Background saving scheduled");
+     * }</pre>
+     */
+    CompletableFuture<String> bgsave(boolean schedule);
+
+    /**
+     * Saves the DataBase in background.
+     *
+     * @see <a href="https://redis.io/commands/bgsave/">redis.io</a> for details.
+     * @param schedule Flag to schedule save on the next opportunity.
+     * @param route Specifies the routing configuration for the command. The client will route the
+     *     command to the nodes defined by <code>route</code>.
+     * @return A server confirmation whether background save started or scheduled.<br>
+     *     An error is returned if there is already a background save running
```

Comment:
I am not sure if we document errors. Pretty sure Sho had a task where she was deleting them pls take a look at node and py.

Reply:
They delete it from tests (validation for error text). It can stay in the doc.

Reply:
No they also deleted it from docs. Please take a look.